app/controllers/users_controller.rb

```diff
@@ -0,0 +1,18 @@
+class UsersController < ApplicationController
+  def index
+    # it'll show all users
+  end
+
```

Comment:
[REQUIRED] Kindly add code for rendering all users to the index action. For example `User.all`.